test/e2e/scenario/logs.scenario.ts

```diff
@@ -104,9 +104,9 @@ describe('logs', () => {
 
       expect(servers.base.app.getLargeResponseWroteSize()).toBeLessThan(
         // When reading the request, chunks length are probably not aligning perfectly with the
-        // response length limit, so it sends few more bytes than necessary. Add a 50% margin of
-        // error to verify that it's still close to the expected limit.
-        DEFAULT_REQUEST_ERROR_RESPONSE_LENGTH_LIMIT * 1.5
+        // response length limit, so it sends few more bytes than necessary. Add a margin of error
+        // to verify that it's still close to the expected limit.
+        DEFAULT_REQUEST_ERROR_RESPONSE_LENGTH_LIMIT * 2
```

Comment:
Why this change?

Reply:
Cf commit message: https://github.com/DataDog/browser-sdk/pull/1405/commits/12f6dd2b745898629185aef0bc83cd4096e50572

Reply:
The comment does not explain why we move from 1.5 to 2. Or am I messing somethingN

Reply:
Maybe I wasn't clear enough? We need a bigger margin of error because apparently with the new wdio packages browsers are reading a bit more data at once (or maybe something in-between the browser and our node server, like the browserstack local tunnel). Let's chat about it on Monday if you want :)

Reply:
Okay! It's clear now ;)